services/QuillLMS/client/app/bundles/Staff/components/evidence/healthDashboards/healthDashboards.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+
+import Navigation from '../navigation'
+import ActivityHealthDashboard from "./activityHealthDashboard";
+import PromptHealthDashboard from "./promptHealthDashboard";
+
+const HealthDashboards = ({ location, match }) => {
+  const [showActivities, setShowActivities] = React.useState<boolean>(true);
+
+  const handleDashboardToggle = () => {
+    setShowActivities(!showActivities)
+  }
+
+  return(
+    <React.Fragment>
+      <Navigation location={location} match={match} />
+      <div className="health-dashboards-index-container">
+        {showActivities ? <ActivityHealthDashboard handleDashboardToggle={handleDashboardToggle} /> : <PromptHealthDashboard handleDashboardToggle={handleDashboardToggle} />}
```

Comment:
this is just a small stylistic suggestion, but I usually opt to pull conditional rendering out into a function for better readability, something like this:

```
function renderDashboard() {
  if(showActivities) {
    return <ActivityHealthDashboard handleDashboardToggle={handleDashboardToggle} />
  }
  return <PromptHealthDashboard handleDashboardToggle={handleDashboardToggle} />
}
```